HW2/index.html

```diff
@@ -1,64 +1,72 @@
-                <a href="#" class="top-menu__item f-p">Functional Programming</a>
-                <a href="#" class="top-menu__item ramda">Ramda</a>
-                <a href="#" class="top-menu__item f-l-s">Fantasy Land Spec</a>
-                <a href="#" class="top-menu__item n-f-a">Node.js for all</a>
-            </nav>
-        </header>
+      <header>
+        <div class="header">
+          <a href="#"
+            ><span class="header-ra">Ra</span
+            ><span class="header-mda">mda.js</span></a
+          >
+        </div>
+        <nav class="top-menu">
+          <a href="#" class="top-menu__item">Functional Programming</a>
+          <a href="#" class="top-menu__item">Ramda</a>
+          <a href="#" class="top-menu__item">Fantasy Land Spec</a>
+          <a href="#" class="top-menu__item">Node.js for all</a>
+        </nav>
+      </header>
 
-        <div class="main-content">
-            <div class="left-menu">
-                <a href="#" class="left-menu__subPart"><span class="left-menu__subPart--ramda">Ramda</span></a>
-                <a href="#" class="left-menu__subPart"><span class="left-menu__subPart--why-ramda">Why Rmada?</span></a>
-                <a href="#" class="left-menu__subPart"><span class="left-menu__subPart--whats-different">What's Different?</span></a>
-                <a href="#" class="left-menu__subPart"><span class="left-menu__subPart--introduction">Introduction</span></a>
-                <a href="#" class="left-menu__subPart"><span class="left-menu__subPart--philosophy">Philosophy</span></a>
-            </div>
-            <div class="sub-content">
-                <div class="top-img">
-                    <img class="logo" src="img/image.JPG" alt="Goat-Image" />
-                </div>
-                <div class="content">
-                    <p class="paragraph">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Beatae
-                        repudiandae neque accusantium nemo assumenda delectus architecto,
-                        eveniet iusto iste sed quo ullam nobis repellendus libero vero
-                        explicabo rem, quis error. Nisi, maiores illo et expedita,
-                        laudantium eaque autem quod delectus vero tenetur nemo accusamus
-                        nostrum explicabo eveniet mollitia illum deleniti.</p>
-                    <p class="paragraph">Lorem ipsum dolor sit amet consectetur adipisicing elit. Saepe,
-                        ipsam? Maiores incidunt, molestiae quo voluptatum, rem tempora earum
-                        debitis praesentium quis autem dolores ipsam explicabo repudiandae!
-                        Consequatur eum consectetur, ex, quasi eius aperiam qui quis a modi
-                        repellendus accusantium omnis architecto illum nesciunt autem nihil
-                        soluta nobis doloribus sapiente corrupti?</p>
-                </div>
-            </div>
+      <main class="main-content">
```

Comment:
Better to use the following example to arrange main-content elements into two columns via flex:

https://stackblitz.com/edit/js-wnh4co?file=index.js,index.html,style.css